googleMapsApiModule.py

```diff
@@ -2,15 +2,17 @@
 import webbrowser
 import config
 import mysql.connector
+from constants import BASE_URL
+
+gmaps = googlemaps.Client(config.key)
+#global variable gmaps
 
 def direction(origin,destination):
-    gmaps = googlemaps.Client(config.key)
     result = gmaps.directions(origin,destination)
     address = "origin="+origin+"&"+"destination="+destination
     address = address.lower()
     address = address.replace(" ","+")
-    url = "https://www.google.com/maps/dir/?api=1&"
-    result_url = url+address
+    result_url = "{}={}".format(BASE_URL['direction'], address.lower().replace(" ", "+"))
```

Comment:
Just 1 minor change. Change `{}={}` to `{}&{}` for this line only, and we're good to go

Reply:
@athityakumar I did what you suggested, but frankly I have never seen that line of statement. Thanks to you who made me familiar with that.

Reply:
@athityakumar  @hritikpapa I thought we were fixing string formatting in another issue. Also, as I had suggested earlier, f-strings are the way to format strings going forward [Python 3's f-Strings: An Improved String Formatting Syntax (Guide)](https://realpython.com/python-f-strings/)

Reply:
@janakrajchadha thanks for pointing this out, this is being fixed in another issue and not the scope of this issue. As mentioned [here](https://github.com/vishakha-lall/MapBot/pull/42#discussion_r387578506), we need to *not* use any formatted string at all, not even ` {}&{}`.
I'm afraid I have to open this PR again.